timemachine/fe/single_topology.py

```diff
@@ -1235,6 +1232,19 @@ def _parameterize_host_guest_nonbonded(
             atom_idxs=np.arange(num_host_atoms, dtype=np.int32),
         )
 
+        return combined_nonbonded.bind(hg_nb_params)
+
+    def _parameterize_host_guest_nonbonded(
```

Comment:
nit: _parameterize_host_guest_nonbonded_ixn to emphasize this is now the ixn only and not the full host-guest potential

Reply:
Renamed in https://github.com/proteneer/timemachine/pull/1087/commits/f38038adf130f0516b958232480abc0f66dbfd1e